src/Raven.Server/Commercial/LicenseValidator.cs

```diff
@@ -34,18 +35,25 @@ private static DateTime FromDosDate(ushort number)
             return new DateTime(year, month, day);
         }
 
-        private static byte[] GetBytes(string str)
+        private static byte[] GetBytesFromBase64String(string str)
         {
             return Convert.FromBase64String(str);
         }
 
+        private static string GetString(byte[] bytes)
```

Comment:
This is wrong on several levels. You should be use the UTF8Enocding for this, instead.